init.lua

```
@@ -28,12 +45,25 @@ local function ensure_packer()
-    logger.info("Packer not found. Cloning packer.nvim...") -- Added logger.info
-    fn.system({
+    logger.info("Packer not found. Cloning packer.nvim...")
+    
+    -- Use security.safe_execute if available, otherwise use fn.system
+    local clone_cmd = {
       'git', 'clone', '--depth', '1',
       'https://github.com/wbthomason/packer.nvim',
       install_path,
-    })
+    }
+    
```

Luacheck (1.2.0)

[warning] line contains only whitespace

(W611)